yogstation/code/modules/admin/verbs/adminhelp.dm

```diff
@@ -428,7 +429,27 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 	log_admin_private(msg)
 	AddInteraction("Marked as WIKI issue by [usr.ckey]")
 	Resolve(silent = TRUE)
+	
+//Resolve ticket with bug message
+/datum/admin_help/proc/GithubIssue(key_name = key_name_admin(usr))
+	if(state != AHELP_ACTIVE)
+		return
+	
+	var/msg = "<font color='red' size='4'><b>- AdminHelp marked as a Github issue by [usr.client.holder?.fakekey ? "an Administrator" : key_name(usr, 0, 0)]! -</b></font><br>"
+	msg += "<font color='red'><b>You are reporting a Bug or Github Issue.</b></font><br>"
+	msg += "<font color='red'>[CONFIG_GET(string/githuburl)]</font>"
+	msg += "<font color='red'><b>Please navigate to the issues section and open up an issue regarding the bug or other issue you have discovered. A maintainer will deal with it shortly.</b></font><br>"
```

Comment:
This is lazy, the issues url is a static addition to the github URL, just direct them directly to it. In fact, should probably direct to the template itself

Reply:
`/issues/new?template=bug_report.md` innit

Reply:
see now and dm me if it doesnt work